src/wrapper/objects/jinputstream.rs

```diff
@@ -0,0 +1,86 @@
+use crate::{
+    errors::*,
```

Comment:
nit: Blob imports are considered bad practice, as it's impossible to see what exactly came from this module.

Reply:
Thanks! Fixed